.shadowenv.d/.gitignore

```diff
@@ -0,0 +1,2 @@
+/.*
+!/.gitignore
```

Comment:
This was auto-added, but it's useful because some error messages are created in that directory and it messes up with git.